src/components/templates/PosterPage/components/ShareModal/ShareModal.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useMemo } from "react";
+import { Modal } from "react-bootstrap";
+import { useCopyToClipboard } from "react-use";
+import classNames from "classnames";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebook, faTwitter } from "@fortawesome/free-brands-svg-icons";
+
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+
+import { PosterPageVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { getFullVenueInsideUrl } from "utils/url";
+
+import { LinkButton } from "components/atoms/LinkButton";
+
+import {
+  getTitleTextForSharing,
+  getFacebookHref,
+  getTwitterHref,
+} from "./helpers";
```

Comment:
[nit, NB] We tend to call these `utils` rather than `helpers`

```suggestion
} from "./utils";
```

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/ce99953a2c8fa07fc9ad3110344481163911e470